string place_name
---
geometry_msgs/Pose pose
bool done
